src/frontend/src/views/GroupMembersPage/style.ts

```diff
@@ -35,9 +36,9 @@ export const StyledHeader = styled.div`
       border-bottom: 2px solid ${colors?.gray[200]};
       padding: 0 ${PAGE_PADDING}px;
 
-      @media only screen and (max-width: 768px) {
```

Comment:
I don't know enough about CSS off hand: should we care about this difference

```
@media only screen and (max-width: 768px)
VS
@media (max-width: 768px)
```

Is it fine for them to all just use the same thing, or are there cases when the `only screen` aspect would be an important differentiator?